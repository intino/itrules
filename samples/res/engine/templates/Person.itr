defrule type(Person)
    $name was born in $country on $birthday+ShortDate
endrule
