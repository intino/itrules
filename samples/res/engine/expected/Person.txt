Pau Gasol was born in Spain on 06/07/1980